src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -308,6 +311,18 @@ private void decodeOtherParameter(Position position, int id, ByteBuf buf, int le
             case 199:
                 position.set(Position.KEY_ODOMETER_TRIP, readValue(buf, length, false));
                 break;
+            case 200:
+                position.set("sleepMode", readValue(buf, length, false));
+                break;
+            case 232:
+                position.set("cngStatus", readValue(buf,length,false) == 1); // CNG Status 1=driving on CNG
```

Comment:
Remove comments for these parameters and add missing spacing in `readValue`. Same for other.